### PR TITLE
Battle-log: Add documentation explaining custom HTML tags

### PR DESCRIPTION
**Pull request by mia-pi-git**

### CODE REVIEW

**Comment by Zarel** [comment]
...this is not exactly the documentation I was asking for.

**Comment by mia-pi-git** [comment]
> ...this is not exactly the documentation I was asking for.

What was it, then?

**Comment by Zarel** [comment]
I was looking for an addendum to `PROTOCOL.md` that described what HTML was usable.

**Comment by mia-pi-git** [comment]
> I was looking for an addendum to `PROTOCOL.md` that described what HTML was usable.

....oh. Okay.

**Comment by Zarel** [comment]
image

Like for this, document what HTML is supported.

**Comment by mia-pi-git** [comment]
Alright. will do. Sorry! Misunderstood.

**Comment by Zarel** [comment]
I mean it's not like this _isn't_ useful. You don't have to close it just because it wasn't what I was asking for.

File: src/battle-log.ts [hunk 1]
```diff
@@ -754,6 +754,17 @@ class BattleLog {
...
+			 * Custom html parsing.
+			 * `tagName` is the input tag, e.g <psicon>.
+			 * To do what you want, you set tagName to the new tag you want it to be replaced with,
+			 * for example changing tagName to <span> for <psicon>.
+			 * You then push the attributes you want (i.e height, width for an <img> tag)
+			 * to the `attribs` array. Caja will remove the old, invalid, attributes.
+			 * Note that you also need to add each attrib to the two Object.assign patterns above.
+			 * Both will require the format `'tagName::attributeName': 0 (or opts, but 0 by default)`.
+			 * Opts include making it an autoclose tag, etc.
+			 */
```
Comment (Zarel) [inline src/battle-log.ts:767]: Doc comments `/** like this */` are specifically meant to document how to use a function, or how to use a variable.

Comments that describe how to _edit_ a function should just use the normal `//` kind of comment.

File: src/battle-log.ts [hunk 2]
```diff
@@ -754,6 +754,17 @@ class BattleLog {
...
+			/**
+			 * Custom html parsing.
+			 * `tagName` is the input tag, e.g <psicon>.
+			 * To do what you want, you set tagName to the new tag you want it to be replaced with,
+			 * for example changing tagName to <span> for <psicon>.
+			 * You then push the attributes you want (i.e height, width for an <img> tag)
+			 * to the `attribs` array. Caja will remove the old, invalid, attributes.
+			 * Note that you also need to add each attrib to the two Object.assign patterns above.
+			 * Both will require the format `'tagName::attributeName': 0 (or opts, but 0 by default)`.
+			 * Opts include making it an autoclose tag, etc.
```
Comment (Zarel) [inline src/battle-log.ts:766]: This seems like a whole lot of information that's sort of just cluttered together. Let me try to find a better way to organize this.

**Pull request closed without merging**